src/test/ui/report-similar-candidates-ordering.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// issue-53302
```

Comment:
It seems like you added this comment after running `./x.py test src/test/ui --bless`:

```
[00:46:07] ---- [ui] ui/report-similar-candidates-ordering.rs stdout ----
[00:46:07] diff of stderr:
[00:46:07] 
[00:46:07] 1	error[E0277]: the trait bound `i32: std::convert::From<&str>` is not satisfied
[00:46:07] -	  --> $DIR/report-similar-candidates-ordering.rs:12:21
[00:46:07] +	  --> $DIR/report-similar-candidates-ordering.rs:14:21
[00:46:07] 3	   |
[00:46:07] 4	LL |     let _: i32 = "".into(); //~ ERROR [E0277]
[00:46:07] 5	   |                     ^^^^ the trait `std::convert::From<&str>` is not implemented for `i32`
```